mealie/services/scraper/cleaner.py

```diff
@@ -431,7 +438,9 @@ def clean_time(time_entry: str | timedelta | None, translator: Translator) -> No
             # TODO: Not sure what to do here
             return str(time_entry)
         case _:
-            logger.warning("[SCRAPER] Unexpected type or structure for variable time_entry")
+            logger.warning(
+                "[SCRAPER] Unexpected type(%s) or structure for variable time_entry: %s", type(time_entry), time_entry
```

Comment:
Let's make this an f-string for readability

```suggestion
                f"[SCRAPER] Unexpected type({type(time_entry)}) or structure for {time_entry=}"
```

Reply:
Ah, i did originally do that, but changed it when pylint raised this warning https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html

it probaly doesnt matter in this usecase

Reply:
Huh, I had no idea this was a thing. Nevermind then, the way you have it is fine, thanks!